pnpm-lock.yaml

```diff
@@ -2376,7 +2376,7 @@ packages:
       react-is: 18.2.0
       rxjs: 7.8.0
       styled-components: 5.3.6(react-dom@18.2.0)(react-is@18.2.0)(react@18.2.0)
-      vite-plugin-environment: 1.1.3(vite@4.2.1)
+      vite-plugin-environment: 1.1.3(vite@4.0.5)
```

Comment:
To endringer lenger opp oppgraderer du fra 4.0.5 til 4.2.1, mens her nedgraderer du fra 4.2.1 til 4.0.5?

Reply:
Approver, antar du vet hva du har gjort

Reply:
Lock filen lever sitt eget liv, ikke gjort noe bevisst her